### DISCUSSION TITLE
How to capitalise first letter?

### DISCUSSION THREAD

**Discussion by multiplehats** [ivanhofer/typesafe-i18n#706] [General]
First of all, thank you. This is a terrific library, and I'm just now getting really in the weeds with it .
What would be the best approach for dealing with sporadic first letter capitalizations? I tried looking at formatters, but I'm not sure if that's the way to do it. 

en:
```
Add your socials
```

nl:
```
Socials toevoegen
```

And I guess this could be a possibility too:

de:
```
Fügen Sie soziale Netzwerke hinzu
```

**Comment by ivanhofer** [comment]
Thanks for your kind words!

Formatters are only applied to `parameters`, so you could transform e.g. dates to a human readable output.

The easiest option is to already capitalize the sentence in your translation file. But depending on where those files come from, you might not have full control over it.

What about doing it in CSS?
```css
text-transform: capitalize;
```
